pyaedt/generic/DataHandlers.py

```diff
@@ -92,41 +100,46 @@ def _dict2arg(d, arg_out):
 
 @aedt_exception_handler
 def _arg2dict(arg, dict_out):
-    """
-
-    Parameters
-    ----------
-    arg :
-
-    dict_out :
-
-
-    Returns
-    -------
-
-    """
     if arg[0] == "NAME:DimUnits" or "NAME:Point" in arg[0]:
-        dict_out[arg[0][5:]] = list(arg[1:])
+        if arg[0][5:] in list(dict_out.keys()):
+            if isinstance(dict_out[arg[0][5:]][0], (list, tuple)):
+                dict_out[arg[0][5:]].append(list(arg[1:]))
+            else:
+                dict_out[arg[0][5:]] = [dict_out[arg[0][5:]]]
+                dict_out[arg[0][5:]].append(list(arg[1:]))
+        else:
+            dict_out[arg[0][5:]] = list(arg[1:])
     elif arg[0][:5] == "NAME:":
         top_key = arg[0][5:]
         dict_in = OrderedDict()
         i = 1
         while i < len(arg):
-            if (type(arg[i]) is list or type(arg[i]) is tuple or str(type(arg[i])) == r"<type 'List'>") and arg[i][0][
-                :5
-            ] == "NAME:":
+            if (isinstance(arg[i], (list, tuple)) or str(type(arg[i])) == r"<type 'List'>") and arg[i][0][
+                                                                                                :5] == "NAME:":
                 _arg2dict(list(arg[i]), dict_in)
                 i += 1
             elif arg[i][-2:] == ":=":
                 if str(type(arg[i+1])) == r"<type 'List'>":
-                    dict_in[arg[i][:-2]] = list(arg[i + 1])
+                    if arg[i][:-2] in list(dict_in.keys()):
+                        dict_in[arg[i][:-2]].append(list(arg[i + 1]))
+                    else:
+                        dict_in[arg[i][:-2]] = list(arg[i + 1])
                 else:
-                    dict_in[arg[i][:-2]] = arg[i + 1]
+                    if arg[i][:-2] in list(dict_in.keys()):
```

Comment:
@maxcapodi78 @beliaev-maksim Could it be modified to
```suggestion
                    if arg[i][:-2] in list(dict_in.keys()):
```

Reply:
not clear suggestion

Reply:
yes, could be